project/src/test/java/com/google/sps/data/UserFeedbackTest.java

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import com.google.common.collect.ImmutableList;
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Optional;
+
+@RunWith(JUnit4.class)
+public final class UserFeedbackTest {
+
+  private static final String PLACE_ID_1 = "12345";
+  private static final ImmutableList<String> PLACES_WITH_PLACE_ID_1 =
+      ImmutableList.of(PLACE_ID_1, "11111", "22222");
+
+  @Test
+  public void build_invalidBothChoosePlaceAndTryagain_throwsIllegalArgumentException() {
+    assertThrows(IllegalArgumentException.class, () -> {
+      UserFeedback.builder()
+          .setPlacesRecommendedToUser(ImmutableList.of(PLACE_ID_1, "11111", "22222"))
```

Comment:
You can use here `PLACES_WITH_PLACE_ID_1`

Reply:
Right, thanks!